11/umbraco-cms/extending/filesystemproviders/azure-blob-storage.md

```diff
@@ -4,7 +4,9 @@ description: Setup your site to use Azure Blob storage for media and ImageSharp
 
 # Using Azure Blob Storage for Media and ImageSharp Cache
 
-For Umbraco sites, there are some scenarios when you may want or need to consider using Azure Blob Storage for your media, particularly if your site contains large amounts of media. Having your site's media in Azure Blob Storage can also help your deployments complete more quickly, and has the potential to positively affect site performance, as the ImageSharp cache is moved to Azure Blob Storage.
+There are some scenarios where you may want or need to consider leveraging Azure Blob Storage. An example would be for media in Umbraco sites with substantial media libraries.
+
+Having your site's media in Azure Blob Storage can help your deployments complete more quickly. This has the potential to positively affect site performance, as the ImageSharp cache is moved to Azure Blob Storage.
```

Comment:
@OwainWilliams Any reason why, you have not kept this line in the V10 documentation?